src/main/java/de/tum/in/www1/artemis/domain/GradingScale.java

```diff
@@ -31,6 +37,12 @@ public class GradingScale extends DomainObject {
     @Column(name = "bonus_strategy")
     private BonusStrategy bonusStrategy;
 
+    @Column(name = "plagiarism_grade")
```

Comment:
I tried setting the values below, it seems both saving and retrieving still worked. Do you know does the `length` enforce a validation check instead of a db error maybe? If not, maybe we can avoid putting them to avoid a potential mismatch with the actual value we have in migration.

```java
@Column(name = "plagiarism_grade" , length = 150)
private String plagiarismGrade;

@Column(name = "no_participation_grade", length = 50)
private String noParticipationGrade;
```

Reply:
I dug a bit deeper, seems like @Size would be even better.
See many usages in our code an https://www.baeldung.com/jpa-size-length-column-differences

Reply:
Yes, together with the `@Valid` annotation you can then perform validation.

Reply:
Added the mentioned annotations in https://github.com/ls1intum/Artemis/pull/5852/commits/9c7c517de74d417883f529aeba50d920eb2ef35a.